runtime/mangata-rococo/src/lib.rs

```diff
@@ -474,6 +486,7 @@ impl pallet_xyk::Config for Runtime {
 	type DisabledTokens = Nothing;
 	type AssetMetadataMutation = AssetMetadataMutation;
 	type WeightInfo = weights::pallet_xyk_weights::ModuleWeight<Runtime>;
+	type RewardsMigrateAccount = RewardsMigrateAccountProvider<Self>;
```

Comment:
please set same seession length && rewareds distribution period in rococo

Reply:
changed to be same